<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Визуальный редактор кривых второго порядка</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.00429185, y2:1, stop:0 rgba(253, 239, 255, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 211, 252, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(255, 121, 218, 145), stop:0.45 rgba(255, 159, 244, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.55 rgba(255, 110, 237, 255), stop:0.562232 rgba(255, 218, 71, 130), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(247, 0, 255, 69), stop:1 rgba(255, 255, 0, 69));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.558, y1:0.528, x2:1, y2:1, stop:0 rgba(255, 229, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>171</width>
           <height>551</height>
          </rect>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Чертежи</string>
          </property>
         </column>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>200</y>
           <width>271</width>
           <height>361</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>0</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Инварианты&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;S=A+C=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;O=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>110</y>
            <width>3</width>
            <height>81</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>3</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>110</y>
            <width>21</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;A &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;B &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>110</y>
            <width>3</width>
            <height>81</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>3</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>110</y>
            <width>21</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;B&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;C &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>140</y>
            <width>21</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Q=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>210</y>
            <width>3</width>
            <height>101</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>3</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="Line" name="line_4">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>210</y>
            <width>3</width>
            <height>101</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>3</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>210</y>
            <width>21</width>
            <height>101</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;B&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;D&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>210</y>
            <width>21</width>
            <height>101</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;B&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;E&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>210</y>
            <width>21</width>
            <height>101</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;D&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;E&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;F&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>240</y>
            <width>21</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="lS">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>58</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lO">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>138</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="lQ">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>240</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>20</y>
           <width>301</width>
           <height>171</height>
          </rect>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QTextEdit" name="textEdit">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>301</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>251</width>
            <height>34</height>
           </rect>
          </property>
          <property name="text">
           <string>Построить новый график </string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>111</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Значение x от:</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="diap1">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>90</y>
            <width>68</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimum">
           <double>-999.990000000000009</double>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>90</y>
            <width>31</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>до:</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="diap2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>90</y>
            <width>68</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimum">
           <double>-999.990000000000009</double>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>141</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Количество точек:</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="toch">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>120</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>301</width>
            <height>171</height>
           </rect>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>80</y>
             <width>221</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Канонический вид уравнения</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>10</y>
             <width>171</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Общий вид уравнения</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>261</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>x+y=1</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>291</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Ax&lt;/span&gt;&lt;span style=&quot; font-size:10pt; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;+2Bxy+Cy&lt;/span&gt;&lt;span style=&quot; font-size:10pt; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;+2Dx+2Ey+F=0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="plot" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="anewch">
   <property name="text">
    <string>Новый чертеж</string>
   </property>
  </action>
  <action name="aneweel">
   <property name="text">
    <string>Новый элемент</string>
   </property>
  </action>
  <action name="adel">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="aris">
   <property name="text">
    <string>Режим множественного отображения</string>
   </property>
  </action>
  <action name="aris2">
   <property name="text">
    <string>Режим индивидуального отображения</string>
   </property>
  </action>
  <action name="achan">
   <property name="text">
    <string>Изменить уравнение</string>
   </property>
  </action>
  <action name="asave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="atat">
   <property name="text">
    <string>Перетаскивание</string>
   </property>
  </action>
  <action name="aang">
   <property name="text">
    <string>Поворот</string>
   </property>
  </action>
  <action name="astan">
   <property name="text">
    <string>Стандартный режим</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
